koku/masu/util/aws/common.py

```diff
@@ -452,6 +455,22 @@ def get_account_names_by_organization(arn, session=None):
     return all_accounts
 
 
+def update_account_aliases(schema, credentials):
+    """Update the account aliases."""
+    _arn = AwsArn(credentials)
+    account_id, account_alias = get_account_alias_from_role_arn(_arn)
+    with contextlib.suppress(IntegrityError), schema_context(schema):
+        AWSAccountAlias.objects.get_or_create(account_id=account_id, account_alias=account_alias)
+
```

Comment:
suggestion: Add a log warning here on the possibility of encountering an `IntegrityError` under normal conditions and future debugging.

Reply:
The reason we suppress the IntegrityError is because the get_or_create can raise it between getting and creating. If it's raised, it means it already exists.